templates/sponsors/sponsor.html

```diff
@@ -1,50 +1,42 @@
 {% set main_class='sponsors' %}
 {% extends "base.html" %}
-{% block title %}Sponsor EMF{% endblock %}
+{% block title %}Sponsor Electromagnetic Field{% endblock %}
 {% block body %}
 
-<h2>Sponsor EMF</h2>
+<h2>Sponsor Electromagnetic Field</h2>
 
-<div class="well">
-    Electromagnetic Field is no longer accepting sponsors for our 2016 event.
-</div>
     <img src="{{ url_for('static', filename='images/sponsor/sponsor_1.jpg')}}"
-            class="img-thumbnail img-responsive" style="float:right; width:320px">
+            class="img-thumbnail img-responsive" style="float:right; width:320px; margin-left: 16px;">
 <p>Electromagnetic Field (EMF) is the UK’s largest technology-focused camping festival: A three-day event for people with an inquisitive mind and an interest in science, engineering, arts, technology, crafts, games, DIY, or security.</p>
 
-<p>Held every two years, EMF is a cross between a conference and a festival, with talks and workshops on a wide range of subjects. EMF is a non-profit event, entirely run by a dedicated team of volunteers who have experience staffing and organising similar events across Europe.</p>
+<p>Held every two years, EMF is a cross between a conference and a festival, with talks and workshops on a wide range of subjects. EMF is a non-profit event, entirely run by a dedicated team of volunteers.</p>
 
-<p>In August 2016 we're running the third EMF festival near Guildford, Surrey. We're providing a programme of three days of fascinating talks, workshops, and performances, to an audience of over 1500 attendees in an internet-connected field.</p>
+<p>At the end of August 2018 we're running the fourth EMF festival at Eastnor Castle Deer Park in Hereforshire. We're providing a programme of three days of fascinating talks, workshops, and performances, to an audience of over 2000 attendees in an internet-connected field.</p>
 
 <h3>Help Us Out</h3>
 <p>Ticket sales provide for the basic EMF infrastructure, but we rely on partnerships with generous sponsors to make the festival a truly amazing place to visit. At previous events, support from sponsors has made the following things possible:</p>
 
 <ul>
-    <li>A custom, <a href="http://blog.emfcamp.org/post/94157161753/announcing-tilda-mke-the-incredible-emf-2014-camp">programmable electronic badge</a> (TiLDA), free to all attendees.</li>
+    <li>A custom, programmable electronic badge (TiLDA), free to all attendees.</li>
     <li>Comprehensive, free on-site childcare and content for young people.</li>
+    <li>Live transcription of talks.</li>
```

Comment:
AFAIK this has not been at any previous events, do we need to change the intro sentence? Maybe to "This year we hope our sponsors will support us in making the following things possible:"